pytket/qir/conversion/conversion.py

```diff
@@ -629,10 +658,9 @@ def condition_block() -> None:
                     )
 
             elif isinstance(op, WASMOp):
-                raise ValueError("WASM not supported yet")
+                raise TypeError("WASM not supported yet")
```

Comment:
I think this is an error in the input and so it should be a value error?

Reply:
I am reverting this change, though I had changed it because it was a failure of `isinstance` check which is about types not values.